Dockerfile

```diff
@@ -14,7 +14,7 @@ FROM build as app-only
 
 EXPOSE 3000
 
-ENTRYPOINT npm start
+ENTRYPOINT npm run start:api:prod
```

Comment:
This broke the docker image as scripts were updated at some point since last release